### ISSUE TITLE
Typo example 18.6, page 336

### ISSUE THREAD

**Issue by bilderbuchi** [sisl/algforopt-errata#9]
There is a typo/formatting error in example 18.6, page 336, where the vector q is given as 
```
q = [0.577, 0.4500.450, 0.417, 0.417]
```
when I think it should read 
```
q = [0.577, 0.450, 0.450, 0.417, 0.417]
```
I did not do the calculations manually to confirm, and sadly have not found time (yet?) to implement the Julia code for the calculations.
As far as I can tell, I have the first printing (at least there is no mention of printing at all).

**Comment by tawheeler** [comment]
This was already fixed in 1st printing errata via 6f9c9ab

**Comment by bilderbuchi** [comment]
ah, ok, great. 
I can't find a mention of this in the errata and the commit sha you give is not in this repo, though.
